couchdbkit/client.py

```diff
@@ -98,16 +95,22 @@ def __init__(self, uri='http://127.0.0.1:5984',
         if uri.endswith("/"):
             uri = uri[:-1]
 
+        url = urlparse(uri)
+        if url.password is not None:
+            uri = url._replace(netloc=url.netloc.rsplit('@', 1)[1]).geturl()
```

Comment:
Oh does urlparse's url type have a funny thing where `url.username` and `url.password` are subsets of `url.netloc`? [Ah, look like it](https://docs.python.org/2/library/urlparse.html#urlparse.urlparse).

Maybe a comment here would help motivate this for the next person who stumbles on it in a few years. Something like "If there's a username and password embedded in the URL, pop them from the URL and pass them as args to `CouchDB`. This makes it less likely that they'll show up in logs that print out the couch server url." I think would be enough to orient someone.